doc/integrator/backend_qgis.rst

```diff
@@ -54,16 +54,8 @@ API.
 
 Your QGIS project layers' CRS should all be in the same CRS. This is subject to change.
 
-Finally, edit your project file manually and set:
-
-.. code::
-
-    <WMSUrl type="QString">https://<host>/mapserv_proxy?ogcserver=<name></WMSUrl>
-    <WFSUrl type="QString">https://<host>/mapserv_proxy?ogcserver=<name></WFSUrl>
-    <WMTSUrl type="QString">https://<host>/mapserv_proxy?ogcserver=<name></WMTSUrl>
-    <WCSUrl type="QString">https://<host>/mapserv_proxy?ogcserver=<name></WCSUrl>
-
-By default, ``https://<host>/mapserv_proxy?ogcserver=<name>`` is empty.
+Finally, in project settings, in QGIS server section set all the published URL to:
```

Comment:
Finally, in the project settings, in the QGIS server section, set all the published URL to: